src/Core/Components/List/FluentPersona.razor

```diff
@@ -1,8 +1,8 @@
 ﻿@namespace Microsoft.FluentUI.AspNetCore.Components
 @inherits FluentComponentBase
 
-<div id="@Id" class="@ClassValue" style="@StyleValue">
-    <div class="initials">
+<div id="@Id" class="@ClassValue" style="@StyleValue" @attributes="@AdditionalAttributes">
+    <div class="initials" @onclick="@OnClick" style="@(OnClick.HasDelegate ? $"cursor: pointer;" : null)">
         <FluentPresenceBadge Status="@Status" Size="@StatusSize" StatusTitle="@StatusTitle">
```

Comment:
Should there be a check on not rendering PresenceBadge when `Status` is `null`

Reply:
That could be done... but no code will be changed because the **Initials** are included in the DIV generated. So, I cannot remove the **Presence Status** code :-(

```html
<div class="fluent-persona">
  <div class="initials">
    <div class="fluent-presence-badge">
	  <div style="display: table-cell; vertical-align: middle; width: 50px; min-width: 50px; height: 50px; min-height: 50px;">
	    LB
	  </div>
	</div>
  </div>
</div>
```